sekoia_automation/connector/__init__.py

```diff
@@ -46,6 +42,46 @@ class Connector(Trigger, ABC):
     # Required for Pydantic to correctly type the configuration object
     configuration: DefaultConnectorConfiguration
 
+    _prometheus_namespace = "symphony_module_common"
+
+    _outcoming_events = Counter(
+        name="forwarded_events",
+        documentation="Number of events forwarded to Sekoia.io",
+        namespace=_prometheus_namespace,
+        labelnames=["intake_key"],
+    )
+
+    _forward_events_duration = Histogram(
+        name="forward_events_duration",
+        documentation="Duration to collect and forward events from eventhub",
+        namespace=_prometheus_namespace,
+        labelnames=["intake_key"],
+    )
+
+    _discarded_events = Counter(
+        name="discarded_events",
+        documentation="Number of events discarded from the collect",
+        namespace=_prometheus_namespace,
+        labelnames=["intake_key"],
+    )
+
+    _events_lag = Gauge(
+        name="events_lags",
+        documentation="The delay, in seconds, from the date of the last event",
+        namespace=_prometheus_namespace,
+        labelnames=["intake_key"],
+    )
```

Comment:
Cannot we move these properties in a mixin class and inherit from this mixin class?